web/src/features/feature-flags/api.ts

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from '@tanstack/react-query';
+import { QUERY_KEYS, getBasePath, getHeaders } from 'api/helpers';
+import { FeatureFlags } from './types';
+
+export async function getFeatureFlags(): Promise<FeatureFlags> {
+  const path = '/feature-flags';
```

Comment:
If this ever changes in the future we only have to change it once.
```suggestion
  const path = `/${QUERY_KEYS.FEATURE_FLAGS}`;
```
Don't forget to add the import.